inst/WORDLIST

```diff
@@ -1,9 +1,32 @@
+args
+brushedPoints
+Closeable
+closeable
+desc
 Draggable
+draggable
+dt
 Forkers
+funder
+ggplot
```

Comment:
@vedhav can you tick out `ggplot` in package documentation so that we do not need to add this to WORDLIST?